src/utils/lodash.js

```diff
@@ -0,0 +1,120 @@
+const Dash = {
```

Comment:
We have a throttle function already that is hand rolled and tiny. We should just add functionality for debounce too if thats what we need.